test/integration/serverTest.js

```diff
@@ -1,29 +1,58 @@
 const chai = require('chai')
 const chaiHttp = require('chai-http')
 const sinonChai = require('sinon-chai')
+const chaiDateTime = require('chai-datetime')
 
 const expect = chai.expect
 const { after, afterEach, before, beforeEach, describe, it } = require('mocha')
 const sinon = require('sinon')
 const { helpers } = require('brave-alert-lib')
-const imports = require('../../index.js')
+const { sleep } = require('brave-alert-lib/lib/helpers')
+const imports = require('../../index')
 
 const db = imports.db
 const redis = imports.redis
 const server = imports.server
-const XETHRU_STATE = require('../../SessionStateXethruEnum.js')
-
-const MOV_THRESHOLD = 17
+const braveAlerter = imports.braveAlerter
+const StateMachine = require('../../stateMachine/StateMachine')
+const XETHRU_STATE = require('../../SessionStateXethruEnum')
+const ALERT_REASON = require('../../AlertReasonEnum')
+
+const MOVEMENT_THRESHOLD = 40
+const STILLNESS_TIMER = 300
+const DURATION_TIMER = 600
+const INITIAL_TIMER = 200
 const IM21_DOOR_STATUS = require('../../IM21DoorStatusEnum')
-const { getRandomArbitrary, getRandomInt, printRandomIntArray } = require('../../testingHelpers.js')
+const { getRandomArbitrary, getRandomInt, printRandomIntArray } = require('../../testingHelpers')
+const STATE = require('../../stateMachine/SessionStateEnum')
 
 chai.use(chaiHttp)
 chai.use(sinonChai)
+chai.use(chaiDateTime)
 
+const sandbox = sinon.createSandbox()
 const testLocation1Id = 'TestLocation1'
 const testLocation1PhoneNumber = '+15005550006'
-const door_coreID = 'door_particlecoreid'
-const radar_coreID = 'radar_particlecoreid'
+const door_coreID = 'door_particlecoreid1'
+const radar_coreID = 'radar_particlecoreid1'
+
+async function firmwareAlert(coreID, alertReason) {
+  try {
+    await chai
+      .request(server)
+      .post('/api/startChatbot')
+      .send({
+        event: `${alertReason} Alert`,
```

Comment:
Ah, I see.

OK, so since we're eventually going to be moving to a new enum for the alert reason when we have the combined Buttons/Sensors DB, what I suggest to avoid the ugly `split` here is  to just create a new enum for `FirmwareAlertType` or something like that which will have the exact two strings that we expect the firmware to send. Then instead of doing a `split` to get the `alertReason` part of it, we would figure out which of the Firmware enum value it is and  based on that , figure out which AlertReason Enum value it should have in our DB. That will give us the most freedom.

But I'm slowing down today in the heat so I'm not sure if I explained that well. Let me know if I didn't make sense!